manifests/bc.yaml

```diff
@@ -0,0 +1,35 @@
+kind: "BuildConfig"
+apiVersion: "build.openshift.io/v1"
+metadata:
+  name: "kubernetes-fcos-pipeline"
+spec:
+  # Define triggers.
+  #   - trigger on buildconfig change
+  #   - trigger on imagestream change
+  #   - trigger on GitHub webhook
+  #   - trigger on Generic webhook (for fedmsg triggered actions)
+  triggers:
+  - type: ConfigChange
+  # use `oc set triggers --from-github` to update token to be unique
+  - type: GitHub
+    github:
+      secret: regenerate
+  # use `oc set triggers --from-webook` to update token to be unique
+  - type: Generic
+    generic:
+      secret: regenerate
+  - type: imageChange
+    imageChange:
+      from:
+        kind: ImageStreamTag
+        name: dustymabe-coreos-assembler:latest
+  source:
+    type: Git
+    git:
+      # XXX: make these into parameters to make it easy for the local dev case
+      uri: 'https://github.com/dustymabe/fedora-coreos-ci'
+      ref: master
+  strategy:
+    jenkinsPipelineStrategy:
+      type: JenkinsPipeline
+      jenkinsfilePath: Jenkinsfile
```

Comment:
does it matter that `Jenkinsfile` is actually one directory up from `manifests/bc.yaml` ?

Reply:
ahh I see it's relative to the uri provided in the source section. 

https://docs.openshift.com/container-platform/3.11/dev_guide/builds/build_strategies.html#pipeline-strategy-options

Reply:
Yup, correct!

I will mention that there is a small downside to splitting them up. That is, you now have to `git push` to test changes to the pipeline. Overall though, I think it's a price worth paying for how much nicer it is to hack on.

Reply:
i guess you could `oc edit` and remove the git link and then `oc apply` local changes and the `oc edit` again to add the git link once you've pushed up changes if you really wanted to ? haven't tested that, but could work in theory.

Reply:
Not sure if I follow. Basically after this PR now:
- for changes to the `bc.yaml` file itself you can do `oc replace -f bc.yaml`
- for changes to the Jenkinsfile, you can do `git push && oc start-build ...`

Reply:
right. yeah I was just describing a flow in which you wanted to test something *without* a `git push`.